### PR TITLE
Added warning to avoid composition problems.

### LINKED ISSUE #136: Add warning for improperly formed schemes using multiple schemas
If you use `allOf` there could be conflicting properties between the schemas, which would be an issue. Add guidance around this.

### PR DESCRIPTION
**Pull request by andresuribe87**
Fixes #136

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/193.html" title="Last updated on Aug 9, 2023, 2:12 PM UTC (5d55a39)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/193/6aae47c...5d55a39.html" title="Last updated on Aug 9, 2023, 2:12 PM UTC (5d55a39)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -786,8 +786,8 @@ <h3>Multiple Schemas</h3>
             }
             </pre>
           </p>
-          <p class="issue" data-number="136">
-            Add warning for improperly formed schemas and risks associated with using multiple schemas.
+          <p>
+            Composition using `allOf` into a JSON Schema can easily result in a schema for which all JSON documents will fail to validate. Such a situation may happen when multiple schemas reference the same property. Schema authors are advised to ensure that there exists at least one JSON document for which schema validation would pass.
```
Comment (decentralgabe) [inline index.html:790]: this looks good - can add a sentence such as ...

"Implementers are advised to test their schemas against a set of sample input documents before introducing any real world usage"?

**Review by decentralgabe — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (andresuribe87) [reply index.html]: Added that and some more.

Comment (decentralgabe) [reply index.html]: thanks!

File: index.html [hunk 2]
```diff
@@ -786,8 +786,8 @@ <h3>Multiple Schemas</h3>
             }
             </pre>
           </p>
-          <p class="issue" data-number="136">
-            Add warning for improperly formed schemas and risks associated with using multiple schemas.
+          <p>
+            Composition using `allOf` into a JSON Schema can easily result in a schema for which all JSON documents will fail to validate. Such a situation may happen when multiple schemas reference the same property. Implementers are advised to test their schemas against a set of sample input documents before introducing any real world usage. Including sample input that suceeds and fails is considered a good practice.
```
Comment (TallTed) [inline index.html:790]:
```suggestion
            Using `allOf` when composing a JSON Schema can easily result in a schema for which all JSON documents will fail to validate. Such a situation may happen when multiple schemas reference the same property. Implementers are advised to test their schemas against a set of sample input documents before introducing any real world usage. Including sample input that suceeds and fails is considered a good practice.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**